packages/react/src/Tooltip/Tooltip.tsx

```diff
@@ -72,19 +72,29 @@ export const Tooltip = React.forwardRef(
     const child = Children.only(children)
     const triggerRef = useProvidedRefOrCreate(forwardedRef as React.RefObject<HTMLElement>)
     const tooltipElRef = useRef<HTMLDivElement>(null)
+    // Used to delay the closing of the tooltip to make sure the user can move the mouse from the trigger to the tooltip
+    const closeTooltipTimeoutRef = useRef<NodeJS.Timeout | null>(null)
+    const tooltipCloseTimeout = 200
 
     const [calculatedDirection, setCalculatedDirection] = useState<TooltipDirection>(direction)
 
     const openTooltip = React.useCallback(() => {
+      if (closeTooltipTimeoutRef.current) {
+        clearTimeout(closeTooltipTimeoutRef.current)
```

Comment:
This is only being cleared programmatically, can/shall we do it in a useEffect too so it runs on unmount too?